docs/code/VBufferCareFeeding.md

```diff
@@ -10,43 +10,32 @@ A `VBuffer<T>` is a generic type that supports both dense and sparse vectors
 over items of type `T`. This is the representation type for all
 [`VectorType`](IDataViewTypeSystem.md#vector-representations) instances in the
 `IDataView` ecosystem. When an instance of this is passed to a row cursor
-getter, the callee is free to take ownership of and re-use the arrays
-(`Values` and `Indices`).
+getter, the callee is free to take ownership of and re-use the buffers, which
+internally are arrays, and accessed externally as `ReadOnlySpan`s through the
+`GetValues` and `GetIndices` methods.
 
-A `VBuffer<T>` is a struct, and has the following `readonly` fields:
+A `VBuffer<T>` is a struct, and has the following most important members:
 
 * `int Length`: The logical length of the buffer.
 
-* `int Count`: The number of items explicitly represented. This equals `Length`
-when the representation is dense and is less than `Length` when sparse.
+* `bool IsDense`: Whether the vector is dense (if `true`) or sparse (if
+  `false`).
 
-* `T[] Values`: The values. Only the first `Count` of these are valid.
+* `ReadOnlySpan<T> GetValues()`: The values. If `IsDense` is `true`, then this
+  is of length exactly equal to `Length`. Otherwise, it will have length less
+  than `Length`.
 
-* `int[] Indices`: The indices. For a dense representation, this array is not
-  used, and may be `null`. For a sparse representation it is parallel to
-  values and specifies the logical indices for the corresponding values. Only
-  the first `Count` of these are valid.
+* `ReadOnlySpan<int> GetIndices()`: For a dense representation, this span will
+  have length of `0`, but for a sparse representation, this will be of the
+  same length as the span returned from `GetValues()`, and be parallel to it.
 
-`Values` must have length equal to at least `Count`. If the representation is
-sparse, that is, `Count < Length`, then `Indices` must have length also
-greater than or equal to `Count`. If `Count == 0`, then it is entirely legal
-for `Values` or `Indices` to be `null`, and if dense then `Indices` can always
-be `null`.
-
-On the subject of `Count == 0`, note that having no valid values in `Indices`
-and `Values` merely means that no values are explicitly defined, and the
-vector should be treated, logically, as being filled with `default(T)`.
-
-For sparse vectors, `Indices` must have length equal to at least `Count`, and
-the first `Count` indices must be increasing, with all indices between `0`
```

Comment:
I think some of this `GetIndices` information is still valuable:

> must be increasing, with all indices between `0` inclusive and `Length` exclusive.

Reply:
Well, I wrote at some length on this subject in the XML docs where I gave more information on the absolute requirements, but a quick mention about the ranges and the fact that it (should be!) sorted is probably something worth repeating, I agree...